opencraft/views.py

```diff
@@ -45,3 +53,46 @@ def get_redirect_url(self, *args, **kwargs):
             return reverse('instance:index')
         else:
             return reverse('registration:register')
+
+class HealthCheckView(View):
+    """
+    Health Check view
+    """
+    def get(self, request):
+        try:
+            # Verify that the postgres database backend is reachable
+            self._check_postgres_connection()
+        except OperationalError:
+            return HttpResponse('Postgres unreachable', status=500)
```

Comment:
Nit: edx-platform uses 503 status for their [heartbeat endpoint](https://github.com/edx/edx-platform/blob/c87df89be5f1d0bda8135253b21590a3b92a738a/openedx/core/djangoapps/heartbeat/views.py#L21), which I think is somewhat more appropriate than 500.